alpine/alpine-unfixed/alpine.go

```diff
@@ -0,0 +1,202 @@
+	for key, version := range baseData {
+		packageData := strings.Split(key, ":")
+		branchPkg := packageData[0]
+		vulnerability := packageData[1]
+		if vulnVersion, exists := branchPkgMap[branchPkg]; exists {
+			if vulns, ok := vulnVersion[version]; ok {
+				vulns = append(vulns, vulnerability)
+				vulnVersion[version] = vulns
+			} else {
+				vulns = []string{vulnerability}
+				vulnVersion[version] = vulns
+			}
+
+		} else {
+			vulns := []string{vulnerability}
+			branchPkgMap[branchPkg] = VulnVersionMap{version: vulns}
+		}
+	}
+	return u.save(branchPkgMap)
+}
+
+func (u Updater) save(packageData map[string]VulnVersionMap) error {
+	for branchPkg, vulnData := range packageData {
+		branchPkgDet := strings.Split(branchPkg, "@@")
+		branch := strings.Split(branchPkgDet[0], "-")
+		release := branch[0]
+		repoName := branch[1]
+		packageName := branchPkgDet[1]
+		dir := filepath.Join(u.vulnListDir, alpineDir, release, repoName)
+		file := fmt.Sprintf("%s.json", packageName)
+		saveJson := SaveJsonFormat{
+			DistroVersion: fmt.Sprintf("v%s", release),
+			RepoName:      repoName,
+			UnfixVersion:  vulnData,
+			PkgName:       packageName,
+		}
+		if err := utils.WriteJSON(u.appFs, dir, file, saveJson); err != nil {
+			return xerrors.Errorf("failed to write %s under %s: %w", file, dir, err)
+		}
+	}
+	return nil
+}
+
+func parseBranchVulnerabilities(branchPath, branch string, baseData map[string]string, baseUrl string) error {
+	b, err := fetchURL(branchPath, "application/ld+json")
+	if err != nil {
+		return err
+	}
+	var releaseInfo ReleaseInfo
+	if err = json.Unmarshal(b, &releaseInfo); err != nil {
+		return err
+	}
+	for _, item := range releaseInfo.Items {
+		_, packageName := path.Split(item.CPEMatch[0].Package)
+		_, vulnerability := path.Split(item.CPEMatch[0].Vulnerability)
+		key := fmt.Sprintf("%s@@%s:%s", branch, packageName, vulnerability)
+		if _, exists := baseData[key]; !exists {
+			err := cveHTMLParser(baseUrl, vulnerability, baseData)
+			if err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+func getActiveReleases(url string) ([]string, error) {
+	var activeRelease []string
+	b, err := fetchURL(url, "text/html")
+	if err != nil {
+		return activeRelease, err
+	}
+
+	d, err := goquery.NewDocumentFromReader(bytes.NewReader(b))
+	if err != nil {
+		return activeRelease, err
+	}
+
+	d.Find("h2").Each(func(i int, selection *goquery.Selection) {
+		activeRelease = append(activeRelease, selection.Text())
+	})
+	return activeRelease, nil
+}
+
+func fetchURL(url, acceptHeader string) ([]byte, error) {
+	req := gorequest.New().Get(url)
+	req.Header.Add("Accept", acceptHeader)
+	resp, body, errs := req.Type("text").EndBytes()
+	if len(errs) > 0 {
+		return nil, xerrors.Errorf("HTTP error. url: %s, err: %w", url, errs[0])
+	}
+	if resp.StatusCode != 200 {
+		return nil, xerrors.Errorf("HTTP error. status code: %d, url: %s", resp.StatusCode, url)
+	}
+	return body, nil
+}
```

Comment:
Expand this function.
https://github.com/aquasecurity/vuln-list-update/blob/main/utils/utils.go#L123-L136

Reply:
Fixed it